lab/04-get-started-with-helm.md

```diff
@@ -0,0 +1,158 @@
+# Helm, the Kubernetes package manager
+
+To this point, we have been using `kubectl` and static YAML manifests to create and delete resources from our Kubernetes cluster. This works well when we have small simple applications. As an application grows in complexity and size you often have 10s of manifests to describe an application. This could include:
+
+* Deployments, Services, configuration ConfigMaps, Secrets
+
+[Helm](https://helm.sh) is a tool that can be installed on any Kubernetes cluster. Helm provides a new concept called a Chart, which groups an arbitrary collection of resources into a logical unit. Let's install `helm` in our cluster:
+
+## Installing Helm
+
+As we did with the `captainkube` application earlier in the lab, let's configure a Service Account and RBAC bindings so Helm has permission to create, modify, and delete resources in our cluster:
+
+```
+kubectl -n kube-system create sa tiller
+kubectl create clusterrolebinding tiller --clusterrole cluster-admin --serviceaccount=kube-system:tiller
+helm init --service-account tiller
+```
+
+**Output**:
+```
+serviceaccount/tiller created
+clusterrolebinding.rbac.authorization.k8s.io/tiller created
+
+Creating /home/jason/.helm
+Creating /home/jason/.helm/repository
+Creating /home/jason/.helm/repository/cache
+Creating /home/jason/.helm/repository/local
+Creating /home/jason/.helm/plugins
+Creating /home/jason/.helm/starters
+Creating /home/jason/.helm/cache/archive
+Creating /home/jason/.helm/repository/repositories.yamlAdding stable repo with URL: https://kubernetes-charts.storage.googleapis.com
+Adding local repo with URL: http://127.0.0.1:8879/charts
+$HELM_HOME has been configured at /home/jason/.helm.
+Happy Helming!
+```
+
+This launches an in-cluster component called `tiller` which interacts with the Kubernetes APIs. We are ready to install some charts.
+
+## Use a chart
+
```

Comment:
I'd like to elaborate a bit here. 

"A chart is a structured collection of yaml files that contain the same information that a manifest might, but are easier to understand, can be templated to remove large amounts of duplicated information and enable dynamic configuration, and can be versioned so as to provide the Helm user with a wide array of sophisticated deployment control."